### PR TITLE
add missing NEMA power ports/outlets

### LINKED ISSUE #4675: add NEMA 1, NEMA 14 power ports/outlets
### Environment
* Python version:  python:3.7
* NetBox version:  2.8.4

### Proposed Functionality
add NEMA 1 and NEMA 14

https://en.wikipedia.org/wiki/File:NEMA_simplified_pins.svg

relate discussion https://github.com/netbox-community/netbox/issues/792

https://github.com/netbox-community/netbox/blob/develop/netbox/dcim/choices.py

### Use Case
uncommon, but some UPS does have NEMA 1-15R outlets

### Database Changes
N/A

### External Dependencies
N/A

### PR DESCRIPTION
**Pull request by ryanmerolle**
### Fixes: #4675 

Corrects or adds the following power outlets and power ports:

- nema-1-15p
- nema-10-30p
- nema-10-50p
- nema-14-20p
- nema-14-30p
- nema-14-50p
- nema-14-60p
- nema-l1-15p
- nema-l5-50p
- nema-l6-15p
- nema-l10-30p
- nema-l14-50p
- nema-l14-60p
- nema-1-15r
- nema-10-30r
- nema-10-50r
- nema-14-20r
- nema-14-30r
- nema-14-50r
- nema-14-60r
- nema-l1-15r
- nema-l5-50r
- nema-l6-15r
- nema-l10-30r
- nema-l14-50r
- nema-l14-60r

### CODE REVIEW

**Comment by ryanmerolle** [comment]
Assuming all looks good I will add to the devicetype-library repo json schema

File: netbox/dcim/choices.py [hunk 1]
```diff
@@ -332,6 +340,12 @@ class PowerPortTypeChoices(ChoiceSet):
...
             (TYPE_NEMA_630P, 'NEMA 6-30P'),
             (TYPE_NEMA_650P, 'NEMA 6-50P'),
+            (TYPE_NEMA_1030P, 'NEMA 10-30P'),
+            (TYPE_NEMA_1050P, 'NEMA 10-50P'),
+            (TYPE_NEMA_1420P, 'NEMA 14-20P'),
+            (TYPE_NEMA_1430P, 'NEMA 14-30P'),
+            (TYPE_NEMA_1450P, 'NEMA 14-50P'),
+            (TYPE_NEMA_1460P, 'NEMA 14-60P'),
         )),
         ('NEMA (Locking)', (
```
Comment (DanSheps) [inline netbox/dcim/choices.py:350]: We should add the locking for the non-locking variants

File: netbox/dcim/choices.py [hunk 2]
```diff
@@ -470,6 +492,12 @@ class PowerOutletTypeChoices(ChoiceSet):
...
             (TYPE_NEMA_630R, 'NEMA 6-30R'),
             (TYPE_NEMA_650R, 'NEMA 6-50R'),
+            (TYPE_NEMA_1030R, 'NEMA 10-30R'),
+            (TYPE_NEMA_1050R, 'NEMA 10-50R'),
+            (TYPE_NEMA_1420R, 'NEMA 14-20R'),
+            (TYPE_NEMA_1430R, 'NEMA 14-30R'),
+            (TYPE_NEMA_1450R, 'NEMA 14-50R'),
+            (TYPE_NEMA_1460R, 'NEMA 14-60R'),
         )),
         ('NEMA (Locking)', (
```
Comment (DanSheps) [inline netbox/dcim/choices.py:502]: Same as above

File: netbox/dcim/choices.py [hunk 3]
```diff
@@ -332,6 +340,12 @@ class PowerPortTypeChoices(ChoiceSet):
...
             (TYPE_NEMA_650P, 'NEMA 6-50P'),
+            (TYPE_NEMA_1030P, 'NEMA 10-30P'),
+            (TYPE_NEMA_1050P, 'NEMA 10-50P'),
+            (TYPE_NEMA_1420P, 'NEMA 14-20P'),
+            (TYPE_NEMA_1430P, 'NEMA 14-30P'),
+            (TYPE_NEMA_1450P, 'NEMA 14-50P'),
+            (TYPE_NEMA_1460P, 'NEMA 14-60P'),
         )),
         ('NEMA (Locking)', (
             (TYPE_NEMA_L515P, 'NEMA L5-15P'),
```
Comment (DanSheps) [inline netbox/dcim/choices.py:351]: We should add the locking variants

File: netbox/dcim/choices.py [hunk 4]
```diff
@@ -470,6 +492,12 @@ class PowerOutletTypeChoices(ChoiceSet):
...
             (TYPE_NEMA_650R, 'NEMA 6-50R'),
+            (TYPE_NEMA_1030R, 'NEMA 10-30R'),
+            (TYPE_NEMA_1050R, 'NEMA 10-50R'),
+            (TYPE_NEMA_1420R, 'NEMA 14-20R'),
+            (TYPE_NEMA_1430R, 'NEMA 14-30R'),
+            (TYPE_NEMA_1450R, 'NEMA 14-50R'),
+            (TYPE_NEMA_1460R, 'NEMA 14-60R'),
         )),
         ('NEMA (Locking)', (
             (TYPE_NEMA_L515R, 'NEMA L5-15R'),
```
Comment (DanSheps) [inline netbox/dcim/choices.py:503]: Same as above

**Review by DanSheps — CHANGES_REQUESTED** [review]

File: netbox/dcim/choices.py (hunk 3 above)
Comment (ryanmerolle) [reply netbox/dcim/choices.py]: I was using a chart I found online for my verification that was missing some L variants.  This has been resolved.  All but the 10-50 have an Locking variant and were added to the choices.

File: netbox/dcim/choices.py (hunk 4 above)
Comment (ryanmerolle) [reply netbox/dcim/choices.py]: same as above

**Review by DanSheps — APPROVED** [review]

**Pull request merged**